api/src/main/java/gov/cms/ab2d/api/controller/v1/BulkDataAccessAPIV1.java

```diff
@@ -64,7 +64,7 @@
 @Slf4j
 @Tag(name = "1. Export", description = BULK_MAIN)
 @RestController
-@RequestMapping(path = API_PREFIX_V1 + FHIR_PREFIX, produces = {APPLICATION_JSON})
+@RequestMapping(path = API_PREFIX_V1 + FHIR_PREFIX, produces = { FHIR_JSON_CONTENT_TYPE, APPLICATION_JSON })
```

Comment:
Is there's a space before and after this. "FHIR_JSON_CONTENT_TYPE, APPLICATION_JSON"? in line 67